src/wikitextprocessor/parserfns.py

```diff
@@ -1155,6 +1159,12 @@ def time_fn(
                     else:
                         delta = add_time - now
                     t = main_date + delta
+        if t is None and orig_dt.isdecimal() and len(orig_dt) == 14:
+            # could be MediaWiki timestamp
+            try:
+                t = datetime.strptime(orig_dt, MEDIAWIKI_TIMESTAMP_FORMAT)
```

Comment:
I haven't use try/except that much, if this throws an exception, what happens to `t`? Will it remain `None`?

Reply:
`t` is still `None` after the `try` block.